tools/testbuild.sh

```diff
@@ -196,12 +201,21 @@ function build {
 # Coordinate the steps for the next build test
 
 function dotest {
-  echo "===================================================================================="
+  if [ ${PRINTLISTONLY} -ne 1 ]; then
```

Comment:
Should we add a function to avoid the code dup here and line 281?

Reply:
do you mean something like this?

    function print_separator {
      if [ ${PRINTLISTONLY} -ne 1 ]; then
          echo "===================================================================================="
      fi
    }

Reply:
Yes.